tests/unit/test_submit.py

```diff
@@ -0,0 +1,84 @@
+import shutil
+from pathlib import Path
+
+import pytest
+
+from cromshell.submit import command as submit_command
+
+
+class TestSubmit:
+    """Test the submit command functions"""
+
+    # Skipping test because function simply holds other functions
+    # def test_validate_input(self, mock_data_path):
+
+    def test_womtool_validate_invalid_wdl_and_json(self, workflows_path):
+        workflow_wdl_path = workflows_path.joinpath("not_valid.wdl")
+        workflow_json_path = workflows_path.joinpath("not_valid.json")
+
+        # asserts that an exception is raised by the function
+        with pytest.raises(Exception):
```

Comment:
should this test be split into four: one for (invalid wdl, valid json), one for (valid wdl, invalid json), and a third for (invalid wdl, invalid json), and lastly one for (valid wdl, valid json), so that no exception is thrown?